src/backend/aspen/workflows/nextstrain_run/export_test.py

```diff
@@ -0,0 +1,98 @@
+import json
+from typing import Iterable
```

Comment:
This is the outline of a new script that allows us to test generating nextstrain build files with different numbers of samples.